proto/storage/cluster.proto

```diff
@@ -94,6 +95,43 @@ message AdmissionControllerConfig {
   bool enforce_on_updates = 5;
 }
 
+message ProcessStatus {
```

Comment:
Status usually refers to the current state or health rather than the configuration setting we want here. I think we should name this message `ProcessConfig` and change the other feature messages to `NetworkConnectionConfig` and `NetworkEndpointConfig`. Future messages which enable new settings could follow the same naming scheme.

Reply:
Done